raiden/tests/integration/network/transport/rif_comms/node.py

```diff
@@ -15,7 +15,7 @@ class Config:
     api_endpoint_prefix = "localhost"
     env_file_prefix = "testing_"
 
-    def __init__(self, node_number: int):
+    def __init__(self, node_number: int, amount_of_clients: int = 1):
```

Comment:
do we really need this default value? if so, why?

Reply:
i think that the default case is 1 client per node

a better option would be to validate amount_of_clients > 0

Reply:
> i think that the default case is 1 client per node
> 
> a better option would be to validate amount_of_clients > 0

i'm sorry, but i don't understand; who is going to call `Config.__init__` without this parameter, and why would we take a defensive approach in such a case?

Reply:
i tought that maybe some tests that dindt use the fixture can create a CommsNode providing a in-place Config object

Reply:
> i tought that maybe some tests that dindt use the fixture can create a CommsNode providing a in-place Config object

correct, but none of these omit the `amount_of_clients` parameter. 

we should also not worry ourselves with arming our own code with tools against its incorrect use; we don't need to do that as long as this is not a public library or something of that sort

Reply:
done